dm_control/mjcf/schema.xml

```diff
@@ -904,6 +904,7 @@
             <attribute name="inflate" type="float"/>
             <attribute name="material" type="reference"/>
             <attribute name="rgba" type="array" array_type="float" array_size="4"/>
+            <attribute name="group" type="int"/>
```

Comment:
Really?! Here?
Please submit this as a separate PR 🙂

Reply:
Oh yes this must've slipped in my branch to get it work, I'll create a separate PR.